<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow &gt; #centralwidget {
	background-image: url(:/wcdo.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0.8);
padding: 10px;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>WcDo Menu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetMenu">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>881</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	background-color: rgba(0, 0, 0, 0.3);
}
QTableWidget::item {
	background-color: rgba(0, 0, 0, 0.3);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlaceOrdrer">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>630</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: white;
border: 2px solid yellow;
font-weight: bold;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Place an Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOrderHistory">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>630</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: white;
border: 2px solid yellow;
font-weight: bold;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>View Orders</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
